sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -93,6 +93,13 @@ def get_regions() -> List[str]:
 # We have to manually remove it.
 DEPRECATED_FAMILIES = ['standardNVSv2Family']
 
+# Some A10 instance types only contains a fractional of GPU. We filter them out
+# here to avoid using it as a whole A10 GPU and causing memory out of capacity.
+# Ref: https://learn.microsoft.com/en-us/azure/virtual-machines/nva10v5-series
```

Comment:
```suggestion
# Some A10 instance types only contains a fractional of GPU. We temporarily
# filter them out here to avoid using it as a whole A10 GPU.
# TODO(zhwu,cbmemo): support fractional GPUs, which can be done on
# kubernetes as well.
# Ref: https://learn.microsoft.com/en-us/azure/virtual-machines/nva10v5-series
```